test/js/upload-wizard/upload-images.test.js

```diff
@@ -0,0 +1,64 @@
+import { screen, fireEvent, waitForElementToBeRemoved } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import _cloneDeep from 'lodash/cloneDeep'
+import selectEvent from 'react-select-event'
+
+import * as ScpApi from 'lib/scp-api'
+
+import {
+  IMAGE_FILE
+} from './file-info-responses'
+
+import { fireFileSelectionEvent } from '../lib/file-mock-utils'
+
+import { renderWizardWithStudy, getSelectByLabelText, saveButton } from './upload-wizard-test-utils'
+
+describe('it allows uploading of reference images', () => {
+  it('uploads a raw counts mtx file', async () => {
```

Comment:
This copy/paste mistake will confuse maintainers.  Possible fix:

```suggestion
  it('validates bad file names, starts upload of JPEG file', async () => {
```